api/envoy/admin/v3/config_dump.proto

```diff
@@ -57,7 +57,9 @@ message ConfigDump {
   // * *clusters*: :ref:`ClustersConfigDump <envoy_v3_api_msg_admin.v3.ClustersConfigDump>`
   // * *endpoints*:  :ref:`EndpointsConfigDump <envoy_v3_api_msg_admin.v3.EndpointsConfigDump>`
   // * *listeners*: :ref:`ListenersConfigDump <envoy_v3_api_msg_admin.v3.ListenersConfigDump>`
+  // * *scoped_routes*: :ref:`ScopedRoutesConfigDump <envoy_v3_api_msg_admin.v3.ScopedRoutesConfigDump>`
   // * *routes*:  :ref:`RoutesConfigDump <envoy_v3_api_msg_admin.v3.RoutesConfigDump>`
+  // * *secrets*:  :ref:`SecretsConfigDump <envoy_v3_api_msg_admin.v3.SecretsConfigDump>`
```

Comment:
Do you happen to know what PRs added support for these additional types?

Reply:
History search indicates that ScopedRouteConfigDump was added in https://github.com/envoyproxy/envoy/pull/6932, and SecretsConfigDump was added in https://github.com/envoyproxy/envoy/pull/7365.